management-interface.go

```diff
@@ -289,6 +292,279 @@ func NetworkHandler(w http.ResponseWriter, r *http.Request) {
 	tmpl.ExecuteTemplate(w, "network.html", state)
 }
 
+//getNetworkSSID -gets the ssid of the network associated with this id
+func getNetworkSSID(networkID string) (string, error) {
+	var out []byte
+	err := error(nil)
+	//get network
+	out, err = exec.Command("wpa_cli", "get_network", networkID, "ssid").Output()
+	if err != nil {
+		return "", fmt.Errorf("error executing wpa_cli get_network %s - error %s output %s", networkID, err, out)
+	}
+
+	stdOut := string(out)
+	scanner := bufio.NewScanner(strings.NewReader(stdOut))
+	scanner.Scan() //skip 1st line interface line
+	scanner.Scan()
+	ssid := scanner.Text()
+	return ssid, err
+
+}
+
+//deleteNetwork - delets network with id using wpa_cli and saves config
```

Comment:
deleteNetwork removes the network from the wpa_supplicant configuration with specified id.